content/docs/install/linux.md

```diff
@@ -1,17 +1,29 @@
 # Installation on Linux
 
-> To use DVC [as a Python library](/doc/api-reference), please
-> [install with pip](#install-with-pip) or [with conda](#install-with-conda).
+<admon type="tip">
+
+To use DVC [as a Python library](/doc/api-reference), please install
```

Comment:
should we have the same note in the mac os?

Reply:
It's there 🙂 See https://dvc-org-install-py-lib-luyi0e0.herokuapp.com/doc/install/macos